addons/l10n_it_edi/tests/test_edi_export.py

```diff
@@ -565,3 +565,28 @@ def test_export_invoice_with_two_downpayments(self):
         )
         invoice.action_post()
         self._assert_export_invoice(invoice, 'test_export_invoice_with_two_downpayments.xml')
+
+    def test_send_prezzo_unitario_converted_to_company_currency(self):
+        """
+        Test that the prezzo unitario is converted to the company currency when the invoice is in a foreign currency
+        """
+        usd = self.env['res.currency'].search([('name', '=', 'USD')])
+        self.env['res.currency.rate'].create({
+            'name': '2025-01-01',
```

Comment:
It doesn't make much sense that the currency rate starts after the invoice date :clock1: